dateparser/custom_language_detection/map_languages.py

```diff
@@ -0,0 +1,17 @@
+from dateparser.data.languages_info import language_map
+
+
+def map_languages(language_codes):
+    """
+    Returns the candidates from the supported languages codes.
+    :param languages:
+        A list of language codes, e.g. ['en', 'es'] in ISO 639 Standard
+    :type languages: list
+    :return: Returns list[str] representing supported languages, else returns None
+    :rtype: list[str]
+    """
+    return_language_codes = []
+    for language_code in language_codes:
+        if language_code in language_map:
+            return_language_codes.extend(language_map[language_code])
+    return return_language_codes
```

Comment:
Hey @noviluni, Its throws an error (`TypeError: unhashable type: 'list'`), as `language_map[language_code]` returns a list.

Reply:
Pl suggest

Reply:
Hi @gavishpoddar, this error doesn't make sense, as the code I wrote is equivalent to what you wrote :thinking: 

this error you mention would make sense if it would be a dictionary, as you can't use a list as a key, but in this case where you are creating a list of lists... Could you check this again? 

Where did you found this error?

Reply:
`language_map[language_code]` returns a list of language codes. So it looks like `[['en']]` (2d array) so the error is raised

Reply:
Hey, I changed it a little bit now it working. Pl suggest if its fine

```
    return [
        language_code
        for language_code_key in language_codes
        if language_code_key in language_map
        for language_code in language_map[language_code_key]
    ]
```

Reply:
Oh! Right, it is an `extend`, not an `append! Sorry, you are right, my code was not equivalent :sweat_smile:  

I took a look to your new code and added a suggestion. Anyways, good job, marking as Resolved :+1: